src/SqlPersistence/obsoletes-v9.cs

```diff
@@ -16,4 +16,28 @@ public partial class SqlPersistence
     }
 }
 
+namespace NServiceBus.Persistence.Sql
+{
+    using System;
+    using System.Linq.Expressions;
+    using Particular.Obsoletes;
+
+    [ObsoleteMetadata(
```

Comment:
Microsoft uses the term "applying an attribute" https://learn.microsoft.com/en-us/dotnet/csharp/advanced-topics/reflection-and-attributes/

Also should we add the code samples between ticks to make it more readable?